akka-persistence-typed/src/main/resources/reference.conf

```diff
@@ -0,0 +1,15 @@
+akka.persistence.typed {
+
+  # Persistent actors stash while recovering or persisting events,
+  # this setting configures the default capacity of this stash.
+  stash-capacity = 2048
+  # If negative (or zero) then an unbounded stash is used (default)
+  # If positive then a bounded stash is used and the capacity is set using
+  # the property
+
+  # enables automatic logging of messages stashed automatically by an PersistentBehavior,
+  # this may happen while it receives commands while it is recovering events or while it is persisting events
+  # Set to a log-level (debug, info, warn, error) to log stashed messages on given log level;
+  log-stashing = off
```

Comment:
We don't have such in other places. It's typical tracing/debugging logging so it makes most sense to log it at debug level. Especially since it `.toString` each message, which is something I don't want anywhere apart from verbose debug.